views/dashboard/about-page/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react'
+import { useSelector } from 'react-redux'
+import FormView from './components/FormView'
+import { StyledAboutWidget } from './StyledAboutWidget'
+
+const AboutWidget = () => {
+    const AboutPageData = useSelector((state: any) => state.aboutPage)
+    const [formView, setFormView] = useState<boolean>(true)
+
+    const handleViewChange = () => {
+        setFormView(!formView)
+    }
+
+    return (
+        <StyledAboutWidget>
+            <div className="header">
+                <h3
+                    className={`${formView && 'active'}`}
+                    onClick={handleViewChange}>
+                    ABOUT ME
+                </h3>
+                <span>/</span>
+                <h3
+                    className={`${!formView && 'active'}`}
+                    onClick={handleViewChange}>
+                    COMPANY STORE
+                </h3>
+            </div>
+            <div className="form-view">
+                {formView ? (
```

Comment:
```
<FormView FormData={formView ? AboutPageData?.[0] : AboutPageData?.[1]}
```

Reply:
Fuck me i did this and was proud but undid it at some point . Cause i originally had 2 sep form view files but we dont talk about that. But that just goes to show my journey to land of mistakenly proud